package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "next-connect-redux",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
I'm pausing a bit about the release number in terms of semver. This PR changes the behaviour by default. Some applications may be relying on the old behaviour (not always getting a fresh store). This means it may need to go in a major release as it's potentially breaking.

Reply:
Thinking about this for a minute, I'm struggling to think up examples of how this would break! And as it's already in `0.x.x` it might be reasonable to just put it in a `0.2.x` release. Curious to see what @huzidaha thinks.